todo/public/bundle.js

```diff
@@ -0,0 +1,126 @@
+/*
```

Comment:
Please don't commit bundle/dist

Reply:
I'll try to deploy app without these folders.